ts/DataGrid/DataGrid.ts

```diff
@@ -518,6 +520,12 @@ class DataGrid {
         }
 
         this.updateGridElements();
+
+        this.gridContainer.ariaLabel = `Grid with ${
```

Comment:
Shouldn't it be configurable?

Reply:
I think that the whole a11y support should be discussed during the DataGrid "new vision". Right now, it is a temporary solution.